code/datums/emotes.dm

```diff
@@ -122,12 +122,6 @@
 				continue
 			if(ghost.client.prefs.chat_toggles & CHAT_GHOSTSIGHT && !(ghost in viewers(user_turf, null)))
 				ghost.show_message("<span class='emote'>[FOLLOW_LINK(ghost, user)] [dchatmsg]</span>")
-	//SKYRAT EDIT ADDITION BEGIN - AI QoL
-	for(var/mob/living/silicon/ai/ai as anything in GLOB.ai_list)
-		var/ai_eye_turf = get_turf(ai.eyeobj)
-		if(ai.client && !(ai.stat == DEAD) && (get_dist(user_turf, ai_eye_turf)<8))
-			ai.show_message("<span class='emote'>[dchatmsg]</span>")
-	//SKYRAT EDIT ADDITION END - AI QoL
```

Comment:
So can AIs still see emotes now?

Reply:
Yes, handled in visible_message() and audible_message() respectively. I did it with the idea so that the AI can actually see the accompanying emotes' runechat text without doing too much snowflake code.